.github/ISSUE_TEMPLATE/regression.md

```diff
@@ -0,0 +1,67 @@
+---
+name: Regression
+about: Report a regression in Rust.
```

Comment:
What about
```suggestion
about: Report a bug for something that used to work but doesn't anymore.
```

Reply:
That seems good. What do other people think?

Reply:
left a [comment](https://github.com/rust-lang/rust/pull/77725#issuecomment-706756620) with a possible alternative

Reply:
Actually just realized that not all regressions are things that used to work but don't anymore: they can also be things that shouldn't work but do. Maybe this?

```suggestion
about: Report something that unexpectedly changed between Rust versions.
```

Cc @Mark-Simulacrum -- what do you think?